README.md

```diff
@@ -18,7 +18,7 @@ Make sure that:
 * You have previously `ssh`-ed into all servers.
   * NOTE: If you are unsure about this, run `bin/sdr check_ssh -e [qa|stage|prod]` and watch the output for any errors!
 * NOTE: if you run `bin/sdr check_cocina`, you may need to ensure that you have the contribsys gem credentials available for google-books to install the sidekiq-pro gem locally (the credential is already on our deploy target VMs).
-  * You can get the env variable name and value from the README in shared_configs for google-books-prod (not in google-books-stage or -qa)
+  * The credentials are set via puppet from values stored in vault.
 * NOTE: You *may* invoke the `bin/` scripts via `bundle exec`.
```

Comment:
I couldn't track down where it is in vault though (or I don't have permissions to see it)

Reply:
looks like i don't have permission either, but the various vault paths to try are explained here (i always have to refer back to this since i use vault infrequently enough to forget between uses):  https://consul.stanford.edu/display/systeam/Vault+for+Developers

might link that doc?

alternatively, you could suggest something more circuitous: look in puppet for the env var name for a VM that uses that value, ssh to the VM, `ksu` or `ssh` to the app user account, print the env var value, and then use that.

Reply:
good suggestions...updated readme